### ISSUE TITLE
AutoField `label` should be honored

### ISSUE THREAD

**Issue by mkilpatrick** [puckeditor/puck#628]
When using AutoField the `label` prop doesn't do anything. You must wrap it in a FieldLabel. The [docs](https://puckeditor.com/docs/extending-puck/custom-fields#rendering-puck-fields-internally) call this out. Either remove `label` from AutoField completely or have it automatically use FieldLabel under the hood.

**Comment by chrisvxd** [comment]
The `AutoField` component doesn't expose a `label` prop, but it does expose a `field.label` prop. Is that what you're referring to?

If so, that is part of the `field` data model, and should be retained. We could call-out that the AutoField will ignore this parameter, though.

**Comment by mkilpatrick** [comment]
Yes, `field.label` is what I was referring to. Could it be honored automatically instead of requiring the FieldLabel wrapper?

**Comment by FedericoBonel** [comment]
This was fixed at some point because the `AutoField` `label` prop type removes the `label` key from it.
